web/concrete/src/Permission/Key/Key.php

```diff
@@ -502,4 +502,10 @@ public static function exportTranslations()
 
         return $translations;
     }
+
+    public static function clearCache()
+    {
+        $cache = Core::make('cache/expensive');
```

Comment:
I don't like this. The Permission Key object shouldn't need any awareness of the application or the cache, we should be invalidating in the controller that adds / deletes keys.